src/main/java/com/google/sps/data/PlayerDatabase.java

```diff
@@ -99,6 +100,13 @@ public String getEntityID() throws LoggedOutException {
     return id;
   }
 
+  // get experience
+  public int getEntityExperience() throws LoggedOutException {
+    int experience =
+        Integer.parseInt(getCurrentPlayerEntity().getProperty(EXPERIENCE_QUERY_STRING).toString());
```

Comment:
Probably not super necessary, but you could encapsulate .parseInt in a try catch exception and return a default value if you catch a NumberFormatException. See here: https://stackoverflow.com/a/18345926